### DISCUSSION TITLE
How to deal with breakpoint in Prometheus data graph

### DISCUSSION THREAD

**Discussion by smallemployees** [prometheus/prometheus#14648] [Q&A]
### What did you do?

A clear breakpoint interval can be seen when querying the database for 30 minutes

### What did you expect to see?

How can I fix this breakpoint situation when the data collection time is 30 seconds

### What did you see instead? Under which circumstances?

A clear breakpoint interval can be seen when querying the database for 30 minutes

### Prometheus version

```text
2.53.1
```

**Comment by machine424** [comment]
Maybe no data was scraped? which query are you running? was the target scraped/`up` during that interval?
